docs/team.rst

```diff
@@ -7,24 +7,103 @@ Team
 TARDIS is developed by a multidisciplinary team. We adhere to the Astropy
 `code of conduct <https://www.astropy.org/code_of_conduct.html>`_.
```

Comment:
Is this still the case after the update or does this sentence need an update too?